src/Splat.Autofac/AutofacDependencyResolver.cs

```diff
@@ -83,26 +83,31 @@ public void SetLifetimeScope(ILifetimeScope lifetimeScope)
         }
 
         /// <inheritdoc />
-        public virtual IEnumerable<object> GetServices(Type serviceType, string? contract = null)
+        public virtual IEnumerable<object> GetServices(Type? serviceType, string? contract = null)
```

Comment:
Remove the nullable on type, it's not allowed to be null anyway.

Reply:
this is a feature already in place, I have expanded the number of resolvers that can use this

Reply:
example `Locator.CurrentMutable.Register(() => foo, null);`